ignition-common-3/PKGBUILD

```diff
@@ -12,10 +12,18 @@ depends=('ignition-math>=6' 'tinyxml2' 'freeimage' 'libutil-linux' 'gts' 'ffmpeg
 makedepends=('ignition-cmake>=2' 'util-linux')
 optdepends=()
 provides=("ignition-common=$pkgver")
```

Comment:
can we make this just `=3` so major version dep can be caught and not just the specific `3.14.0` version.